browser/components/pages/LoggedInHomePage.js

```diff
@@ -1,26 +1,85 @@
 import React, { Component } from 'react'
+import Link from '../atoms/Link'
 import Button from '../atoms/Button'
 import Layout from '../molecules/Layout'
 import InspectObject from '../utils/InspectObject'
-import PendingPrrrs from '../molecules/PendingPrrrs'
-import ClaimedPrrrs from '../molecules/ClaimedPrrrs'
+import claimPrrr from '../../actions/claimPrrr'
+import completePrrr from '../../actions/completePrrr'
+import unclaimPrrr from '../../actions/unclaimPrrr'
+import GithubUsername from '../atoms/GithubUsername'
+import Date from '../atoms/Date'
 
 export default class LoggedInHomePage extends Component {
   render(){
     const { session, prrrs=[] } = this.props
-    return <Layout className="HomePage" session={session}>
 
-      <h1>Pull Requests Waiting For Review:</h1>
-      <PendingPrrrs
-        currentUser={session.user}
-        prrrs={prrrs}
-      />
-
-      <h1>Claimed Pull Requests:</h1>
-      <ClaimedPrrrs
-        currentUser={session.user}
-        prrrs={prrrs}
-      />
+    const unclaimedPrrrs = prrrs.filter(prrr => !prrr.claimed_by)
+
+    const claimedPrrr = prrrs.find(prrr =>
+      prrr.claimed_by === session.user.github_username
+    )
+
+    const infoBox = claimedPrrr
+      ? <ClaimedPrrr prrr={claimedPrrr} />
+      : <ClaimAPrrr />
+
+
+    return <Layout className="HomePage" session={session}>
+      <h1>There are {unclaimedPrrrs.length} Pull Requests ready for Review:</h1>
+      {infoBox}
     </Layout>
   }
 }
+
+
+class ClaimedPrrr extends Component {
```

Comment:
I agree. in an ideal world we would change

```js
const infoBox = claimedPrrr
  ? <ClaimedPrrr prrr={claimedPrrr} />
  : <ClaimAPrrr prrrs={prrrs} session={session}/>
```

into something like:

```js
<ActionBox prrrs={prrrs} />
```

and move move it into its own component for simplicity sake